src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -60,9 +60,12 @@ public static partial class AppContext
         public static string BaseDirectory { get { return default(string); } }
         public static void SetSwitch(string switchName, bool isEnabled) { }
         public static event System.UnhandledExceptionEventHandler UnhandledException { add { } remove { } }
+        public static event System.EventHandler<System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs> FirstChanceException { add { } remove { } }
```

Comment:
i thought this is part of netstandard 1.7 and not specific to netcoreapp1.1. netstandard repo has the api as part of it.